tools/traffic/workers/blob_reader.go

```diff
@@ -0,0 +1,223 @@
+package workers
+
+import (
+	"context"
+	"crypto/md5"
+	"fmt"
+	"github.com/Layr-Labs/eigenda/api/clients"
+	contractEigenDAServiceManager "github.com/Layr-Labs/eigenda/contracts/bindings/EigenDAServiceManager"
+	"github.com/Layr-Labs/eigenda/core"
+	"github.com/Layr-Labs/eigenda/encoding"
+	"github.com/Layr-Labs/eigenda/retriever/eth"
+	"github.com/Layr-Labs/eigenda/tools/traffic/config"
+	"github.com/Layr-Labs/eigenda/tools/traffic/metrics"
+	"github.com/Layr-Labs/eigenda/tools/traffic/table"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	gcommon "github.com/ethereum/go-ethereum/common"
+	"math/big"
+	"sync"
+	"time"
+)
+
+// BlobReader reads blobs from a disperser at a configured rate.
```

Comment:
reads blobs from _the DA network_?

Reply:
Good catch. The code is reading from the DA nodes, the comment was wrong. Fixed.